browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -89,6 +89,7 @@ class AdblockCnameResolveHostClient : public network::mojom::ResolveHostClient {
     network::mojom::ResolveHostParametersPtr optional_parameters =
         network::mojom::ResolveHostParameters::New();
     optional_parameters->include_canonical_name = true;
+    optional_parameters->initial_priority = net::RequestPriority::HIGHEST;
```

Comment:
but how do we avoid stale lookups?

Reply:
Oops, this went through a couple of revisions in conversation with @bridiver and I didn't update the title of the PR in the meantime. We used to have `optional_parameters->cache_usage = network::mojom::ResolveHostParameters::CacheUsage::STALE_ALLOWED;`, but we wanted to test that more.